src/excmds.ts

```diff
@@ -3281,30 +3281,61 @@ export async function yankimage(url: string): Promise<void> {
 
 /** Change active tab.
 
-    @param index
-        Starts at 1. 0 refers to last tab of the current window, -1 to penultimate tab, etc.
+    @param id
+        A string following the following format: "[0-9]+.[0-9]+", the first number being the index of the window that should be selected and the second one being the index of the tab within that window.
+
+        A bare number means the current window is used.
 
         "#" means the tab that was last accessed in this window
 
-    This is different from [[taball]] because `index` is the position of the tab in the current window.
+        A non integer string means to search the URL and title for matches
  */
 //#background
-export async function tab(index: number | "#") {
-    return tabIndexSetActive(index)
+export async function tab(id: string) {
```

Comment:
This should probably be `...id: string[]` so e.g. `:tab BBC news` works and doesn't silently drop the news